lib/config/pacote.js

```diff
@@ -2,13 +2,16 @@
 
 const Buffer = require('safe-buffer').Buffer
 
+const crypto = require("crypto")
 const npm = require('../npm')
 const log = require('npmlog')
 let pack
 const path = require('path')
 
 let effectiveOwner
 
+const npmSession = crypto.randomBytes(8).toString('hex')
```

Comment:
Previously npm-registry-client verbosely logged the value in initialize. As written this loses that (value isn't logged).

Is there a similar place I can generate and verbosely log the value - e.g. is https://github.com/npm/npm/blob/39495d07b9a66c88621e8a2ad07739ee98b70a56/lib/install/action/fetch.js called just once and would it be appropriate to generate npmSession and log it there?